src/profiling_data.rs

```diff
@@ -0,0 +1,17 @@
+use crate::event::Event;
+use std::path::Path;
+
+pub struct ProfilingData {}
+
+impl ProfilingData {
+    pub fn new(_path_stem: &Path) -> ProfilingData {
+        unimplemented!()
+    }
+
+    pub fn iter_events<'a, F>(&'a self, mut _f: F)
```

Comment:
I'd like to see this return an iterator but that's not really important right now.